package.json

```diff
@@ -52,8 +53,8 @@
     "@testing-library/react": "^10.0.2",
     "@types/jest": "^25.1.2",
     "@types/raf": "^3.4.0",
-    "@types/react": "^16.9.19",
-    "@types/react-dom": "^16.9.5",
+    "@types/react": "^17.0.3",
+    "@types/react-dom": "^17.0.3",
```

Comment:
These expose `flushSync`. However, the method existed in 16.8 too (which is your minimal version). It just wasn't in the types.